packages/forms-web-app/src/pages/projects/project-information/utils/pre-examination-sub-stages.js

```diff
@@ -0,0 +1,37 @@
+const dayjs = require('dayjs');
+
+function getPreExaminationSubStage(openDate, closedDate, websiteDate, rule6 = false) {
+	const dayToday = dayjs();
+	const dayOpenDate = dayjs(openDate);
+	const dayClosedDate = dayjs(closedDate);
+	const dayWebsiteDate = dayjs(websiteDate);
+
+	const subStages = {
+		PRE_REPS: false,
+		OPEN_REPS: false,
+		CLOSED_REPS: false,
+		PUBLISHED_REPS: false,
+		RULE_6_PUBLISHED_REPS: false
+	};
+
+	if (openDate === null) subStages.PRE_REPS = true;
+	if (dayToday.isBefore(dayOpenDate)) subStages.PRE_REPS = true;
+
+	if (dayToday.isAfter(dayOpenDate) && (dayToday.isBefore(dayClosedDate) || closedDate === null))
+		subStages.OPEN_REPS = true;
+
+	if (dayToday.isAfter(dayClosedDate) && dayToday.isBefore(dayWebsiteDate))
+		subStages.CLOSED_REPS = true;
+
+	if (rule6) {
+		if (dayToday.isAfter(dayWebsiteDate) && rule6) subStages.RULE_6_PUBLISHED_REPS = true;
+	} else {
+		if (dayToday.isAfter(dayWebsiteDate)) subStages.PUBLISHED_REPS = true;
```

Comment:
"PRE_REPS":false,"OPEN_REPS":true,"CLOSED_REPS":false,"PUBLISHED_REPS":true,"RULE_6_PUBLISHED_REPS":false

Possible with dates -  Open (2017-02-14) closed (2024-08-01) websote (2021-08-17) false

Rep open and published both true

Reply:
test is fine. But when rendered it shows true.